audits/manifest/exists.js

```diff
@@ -47,7 +47,9 @@ class ManifestExists extends Audit {
    */
   static audit(artifacts) {
     return ManifestExists.generateAuditResult(
-      typeof artifacts.manifest !== 'undefined'
+      typeof artifacts.manifest.value !== 'undefined',
+      undefined,
+      artifacts.manifest.warning
```

Comment:
I chose `warning` for the manifest parser because that's the language the spec uses ("issue a developer warning"), but `errorString` or `debugString` seems more appropriate for the audit output. `debugString` works for me in this case.